WebRoot/css/login.css

```diff
@@ -43,6 +43,46 @@ FORM {
 	padding:0px;
 }
 
+.signIn {
+	font-size: 20px;
+	text-align: left;
```

Comment:
We can remove it as it inherited from `#zLoginForm`

Reply:
Done